Dockerfile

```diff
@@ -1,13 +1,18 @@
 # Build the manager binary
-FROM golang:1.13.11 as builder
+FROM golang:1.13.15 as builder
 
 # Install FDB
 ARG FDB_VERSION=6.2.22
 ARG FDB_ADDITIONAL_VERSIONS="6.1.13"
 ARG FDB_WEBSITE=https://www.foundationdb.org
 
 COPY foundationdb-kubernetes-sidecar/website/ /mnt/website/
+
+# FIXME: Workaround for (https://github.com/FoundationDB/fdb-kubernetes-operator/issues/252#issuecomment-643812649)
+# download GeoTrust_Global_CA.crt during install and remove it afterwards
 RUN set -eux && \
+    curl -sLo /usr/local/share/ca-certificates/GeoTrust_Global_CA.crt https://www.geotrust.com/resources/root_certificates/certificates/GeoTrust_Global_CA.pem && \
```

Comment:
Rather than downloading the CA at build-time, can we add it to our repo and copy it into the build? I think pinning it that way will be more secure.